File: DataShare_Web/src/app/core/models/FileMetaDataResponse.ts
export interface FileMetaDataResponse {
  originalFileName: string,
  fileSize: string,
  extension: string,
  token: string,
  createdAt: string,
  expirationDays: number,
  isExpired: boolean,
  isProtected: boolean,
  remainingDays: number,
  expirationDate: string,
  tags: string
}

File: DataShare_API/DTO/FileMetaDataResponse.cs
namespace DataShare_API.DTO
{
    public class FileMetaDataResponse
    {
        public string OriginalFileName { get; set; } = string.Empty;
        public string Token { get; set; } = string.Empty;
        public string Extension { get; set; } = string.Empty;
        public string FileSize { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; } = DateTime.MinValue;
        public int ExpirationDays { get; set; } = 7;
        public bool IsExpired { get; set; } = false; 
        public bool IsProtected { get; set; } = false;
        public int RemainingDays { get; set; } = 0;
        public DateTime ExpirationDate { get; set; } = DateTime.MinValue;
        public string[] Tags {  get; set; } = new string[0];

    }
}